GOVERNANCE.md

```diff
@@ -123,12 +123,12 @@ Beckn protocol specification adopts a set of design principles that must be adhe
 
 
 ### Licensing
-Beckn protocol is currently covered under a Creative Commons License. Just like the design principles, the licensing also may change basis the extent of adoption by the community. The Core Working Group must:
+Beckn protocol is currently covered under a Creative Commons License. Just like the design principles, the licensing also may change the basis of the extent of adoption by the community. The Core Working Group must:
```

Comment:
I would suggest changing this to "the license of Beckn protocol speification may change in the future depending on the scale of its adoption by the community.